src/components/PickupBox/PickupBox.jsx

```diff
@@ -50,25 +50,32 @@ const PickupBox = ({
   const approvePickupBox = async id => {
     const user = await getCurrentUser(auth);
     const userInDB = await FYABackend.get(`/users/userId/${user.uid}`);
+
+    // Retrieve date of most recent transaction.
+    const transaction = await FYABackend.get(`/boxHistory/mostRecentTransaction/${boxID}`);
+
     await FYABackend.put('/boxHistory/update', {
       transactionID: id,
       boxID,
       status: 'evaluated',
       approved: true,
       admin: `${userInDB.data.user.first_name} ${userInDB.data.user.last_name}`,
     });
+    if (Date.parse(transaction.data[0].mostrecentdate) <= Date.parse(date)) {
+      // TODO: REPLACE US WITH COUNTRY INPUT
+      let coordinates = await getLatLong(zipCode, country || 'US');
+      if (coordinates.length !== 2) {
+        coordinates = [0, 0];
```

Comment:
Can we set coordinates to null? This way, we can make sure the box doesn't get added to the map because (0,0) is the wrong location for this box.